components/EducationCarousel.md

```diff
@@ -6,7 +6,7 @@ name: EducationCarousel
 
 A carousel to display educational information on how to succeed with the channel.
 
-Related Polaris components: [Card](https://polaris.shopify.com/components/structure/card)
+Related Polaris components: [Card](https://polaris.shopify.com/components/layout-and-structure/legacy-card)
```

Comment:
```suggestion
Related Polaris components: [Card](https://polaris.shopify.com/components/layout-and-structure/card)
```
This redirects to alpha card and means less update work for us in the future when legacy card is gone